projects/engine/doc/tutorial/2-plugin-communication.md

```diff
@@ -0,0 +1,134 @@
+## Plugins Communication
+
+### Methods
+
+Each plugin can call methods exposed by other plugin. Let's see how to expose a method from one plugin and call it from another.
+
+1. Create Plugin that exposes a method
+You can extends the `Plugin` class to create your own plugin. The list of exposed methods are defined in the field `methods` of the profile: 
+```typescript
+class FirstPlugin extends Plugin {
```

Comment:
I mean that ^

> sounds that this can also be useful for plugin developer?

> Not really. There is a big problem of naming with the remix-plugin right now because :
> Plugin is the code that runs in the engine
> Client is the code that run in the plugin (build by the plugin developer)
> In this case we are talking about the Plugin in the engine.